wallet-new/integration/Functions.hs

```diff
@@ -21,10 +21,11 @@ import           Cardano.Wallet.API.V1.Types (Account (..), AccountIndex, Accoun
                                               PaymentDistribution (..), PaymentSource (..),
                                               SpendingPassword, Transaction (..), V1 (..),
                                               Wallet (..), WalletAddress (..), WalletId,
-                                              WalletOperation (..), WalletUpdate (..))
+                                              WalletOperation (..), WalletUpdate (..), unV1)
 
 import           Cardano.Wallet.API.V1.Migration.Types (migrate)
-import           Cardano.Wallet.Client (ClientError (..), WalletClient (..), getTransactionIndex, getAddressIndex, getAccounts, getWallets)
+import           Cardano.Wallet.Client (ClientError (..), WalletClient (..), getAccounts,
+                                        getAddressIndex, getTransactionIndex, getWallets)
```

Comment:
Judging from these line-wrapped imports, looks like you are -- which means this code was merged without the import formatted, it seems 🤔